htdocs/core/boxes/box_birthdays.php

```diff
@@ -91,7 +91,7 @@ public function loadBox($max = 20)
 			$sql .= " WHERE u.entity IN (".getEntity('user').")";
 			$sql .= " AND u.statut = 1";
 			$sql .= dolSqlDateFilter('u.birth', 0, $tmparray['mon'], 0);
-			$sql .= " ORDER BY u.birth ASC";
+			$sql .= " ORDER BY DAYOFMONTH(u.birth) ASC";
```

Comment:
DAYOFMONTH may not be compatible with other databases than mysql.
Can you try with 
DAY()
instead ?  (this function is added as a custom sql method so it is available for postgresql too)

Reply:
good call and thanks for the update!